search/processor.cpp

```diff
@@ -595,19 +595,21 @@ void Processor::Search(SearchParams const & params)
 
     try
     {
-      SearchDebug();
-      SearchCoordinates();
-      SearchPlusCode();
-      SearchPostcode();
-      if (viewportSearch)
+      if (bool coords_found = SearchCoordinates(); !coords_found)
```

Comment:
```suggestion
      if (!SearchCoordinates())
```

Reply:
"5 7" are coordinates. Additional search results from screenshot are relevant or not?
"-" is skipped in search processing as a delimiter